packages/block-library/src/comments-query-loop/block.json

```diff
@@ -7,21 +7,17 @@
 	"description": "An advanced block that allows displaying post comments based on different query parameters and visual configurations.",
 	"textdomain": "default",
 	"attributes": {
-		"queryId": {
-			"type": "number"
-		},
-		"queryPerPage": {
+		"perPage": {
 			"type": "number",
-			"default": 50
+			"default": null
```

Comment:
It looks like the best way to replicate the behavior from the Query Loop block:

https://github.com/WordPress/gutenberg/blob/8aa82304adbc905564e37ff75795a78a3ff6b4a4/packages/block-library/src/query/block.json#L16-L18

This way we still can use the site settings set for comments. The only issue I discovered is that once we change the value in the UI, it's impossible to reset it with the existing toolbar control.

Reply:
Would it make sense to add an option to inherit the `perPage` value from the WordPress settings? Something similar to the Layout settings, but in the toolbar instead of the sidebar:

![Screenshot 2021-12-09 at 19 39 05](https://user-images.githubusercontent.com/6917969/145456242-59d02150-9b63-414a-b604-fad02cb6b8fd.png)

![Screenshot 2021-12-09 at 19 39 42](https://user-images.githubusercontent.com/6917969/145456255-2a81e791-a585-4159-8d08-69af91841bd6.png)

Reply:
Oh, I see, that would be nice. I feel like it needs like a "(?)" disclaimer in the UI or something like that to clarify what we mean by "inherit" 🙂 

Oh, and we could "link" the `inherit` toggle and the `perPage` selector in javascript so that they they are more intuitive:
- By default the `inherit` is ON.
- Toggle the `inherit` OFF, it sets to the `perPage` to a pre-defined value (e.g `10`).
- Toggle the `inherit` ON, it sets the `perPage` selector to `null`.

Reply:
@DAreRodz, I like the proposal very much, and as far as I remember it's how the Query Loop block operates. @ntsekouras, is that still true?

@c4rl0sbr4v0, so that might be the solution for using the default site setting for everything related to comments. In that scenario, we would have to make all related attributes either don't define any default value or use `null` as an explicit way to tell blocks to read site settings.

Reply:
>@DAreRodz, I like the proposal very much, and as far as I remember it's how the Query Loop block operates. @ntsekouras, is that still true?

Yes it is and when we `inherit` in `Query Loop` we also hide some controls (maybe we should hide more though 😄 ). Another thing in Query Loop is that it initializes the `perPage` from settings if not explicitly provided.